Sources/SBTUITestTunnelClient/SBTUITestTunnelClient.m

```diff
@@ -789,10 +789,16 @@ - (BOOL)coreLocationStubLocationServicesEnabled:(BOOL)flag
 }
 
 - (BOOL)coreLocationNotifyLocationUpdate:(NSArray<CLLocation *>*)locations
+{
+    return [self coreLocationNotifyLocationUpdate:locations dispatchOnMainThread: NO];
+}
+
+- (BOOL)coreLocationNotifyLocationUpdate:(NSArray<CLLocation *>*)locations dispatchOnMainThread:(BOOL) dispatchOnMainThread
```

Comment:
I would keep the API as simple as possible and just always dispatch on main thread instead. Alternatively what about dispatching on main by default?

Reply:
No problem, I just wanted to avoid to introduce breaking changes for this specific fix/workaround.